packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -245,12 +256,15 @@ class VirtualListBase extends Component {
 		spacing: PropTypes.number,
 
 		/**
-		 * TBD
+		 * JS or Native type
```

Comment:
I don't have a better idea yet, but I believe we should refine this explanation later. :) Let's think about this together.